plugin/src/main/java/io/jenkins/plugins/casc/core/AdminWhitelistRuleConfigurator.java

```diff
@@ -56,8 +56,13 @@ public String getName() {
 
     @Override
     protected AdminWhitelistRule instance(Mapping mapping, ConfigurationContext context) {
-        Injector injector = Jenkins.getInstance().getInjector();
-        return injector.getInstance(AdminWhitelistRule.class);
+        Injector injector = Jenkins.get().getInjector();
```

Comment:
I wonder why injector is marked `@CheckForNull`, is there a better way of getting the `AdminWhitelistRule` instance?